frontend/util.lua

```diff
@@ -1034,6 +1034,23 @@ function util.partialMD5(filepath)
     return update()
 end
 
+function util.writeToFile(data, filepath, directory_updated)
+    if not filepath then return end
+    local ok
+    local file = io.open(filepath, "w")
+    if file ~= nil then
+        file:write("-- we can read Lua syntax here!\nreturn ")
+        file:write(data)
+        file:write("\n")
```

Comment:
Feels like naming it `writeToFile` would mean it's really generic, while it indeed specifically writes the provided stuff embedded in that `return...` so it can be later read back with `pcall(dofile(...))`.

So, either give it a less generic name `writeLuaToFile` (or a better name) ?
Or make it accept another option `is_lua_dump_data` (or a better name) and do that prefix/suffix stuff adding only if requested, so this `util/writeToFile()` function can be used for other non-Lua stuff writting too ?

May be we could also update that comment `-- we can read Lua syntax here!` white we're at it - it still always look strange to me when I look at these files :).
`-- KOReader Lua data dump` ?

Reply:
I think it looks enthusiastic. :-P

Reply:
`-- Thank you for looking at me!` then :)

Reply:
It can be the filepath in the comment.

Reply:
People may backup or share their settings or profiles, this line reminds where to place the file to.

Reply:
Yes, where it is from, and what it is (history.lua, settings.reader.lua, metadata.epub.lua, gestures.lua...)